timetracker/entries/migrations/0001_initial.py

```diff
@@ -1,25 +1,51 @@
 # -*- coding: utf-8 -*-
```

Comment:
Correct. Django / your database needs to know to which author it should refer for existing rows. You could either set this to a hard coded user (not the best idea) or to accept that the value might be `None` for existing row, in which case you'd need to set `null=True` in your models. However, if it's `None`, your filtering on users is a bit more advanced. See the solution and the brief discussion in https://github.com/MelbDjango/lesson-eight/pull/5#discussion_r39130180